tests/tests_perf/test_perf_base.py

```diff
@@ -94,6 +94,9 @@ def dpnp_benchmark(self, name, lib, dtype, size, *args, custom_fptr=None, **kwar
         else:
             examine_function = custom_fptr
 
+        # warming up
+        result = examine_function(*args, **kwargs)
```

Comment:
I would vote to avoid warming-up procedure. This makes us blind in performance assessment.
"madian" value has weak dependency on warmup numbers.